### ISSUE TITLE
Button hover contrasts

### ISSUE THREAD

**Issue by Novally** [UTCWeb/particle#110]
Compared to lighter buttons, the colored buttons have very little contrast when the mouse hovers over it.

![image](https://user-images.githubusercontent.com/54911412/96619527-48e0f080-12d4-11eb-807c-c0b4d19bff7b.png)
(Hovering over Blue button)

![image](https://user-images.githubusercontent.com/54911412/96619605-61510b00-12d4-11eb-9f98-dce7922ba47d.png)
(Hovering over Blue outline button)

![image](https://user-images.githubusercontent.com/54911412/96619868-ad03b480-12d4-11eb-805a-32333c45b174.png)
(Hovering over white outline button)

Buttons that have low contrasts: **UTC Green, UTC Blue**
![image](https://user-images.githubusercontent.com/54911412/96620923-04565480-12d6-11eb-8f13-bc3437c53316.png)
![image](https://user-images.githubusercontent.com/54911412/96620970-0b7d6280-12d6-11eb-99ef-2a9f8da7813a.png)
![image](https://user-images.githubusercontent.com/54911412/96621004-159f6100-12d6-11eb-9e8c-af7931bc2350.png)
![image](https://user-images.githubusercontent.com/54911412/96621020-1afcab80-12d6-11eb-9c58-ff9458e8aa59.png)

### Possible solutions:
Can make it the opposite to the light buttons - where the dark blue or yellow will switch to white.  
Or switch the hover color to a light blue and switch the text to a dark blue.  Similar to how the White outlined button switches to yellow with dark blue text.

**Comment by brittrich** [comment]
Agreed, the contrast is somewhat low. The idea of doing an even lighter hover and changing the text color would work, I can take a look. I know in the redesign phase of things this will be something to look at again, too.

**Comment by ghost** [comment]
![Web 1920 – 1](https://user-images.githubusercontent.com/8737117/97342388-8b6a7600-185c-11eb-8082-93038eb59a95.jpg)

**Comment by ghost** [comment]
[button-color-contrast-specs.pdf](https://github.com/UTCWeb/particle/files/5447161/button-color-contrast-specs.pdf)

**Comment by ghost** [comment]
![utc-button-contrast@2x](https://user-images.githubusercontent.com/8737117/97344756-9541a880-185f-11eb-9662-1193ade186a5.jpg)
[button-color-contrast-specs-v2.pdf](https://github.com/UTCWeb/particle/files/5447294/button-color-contrast-specs-v2.pdf)
